{{ cookiecutter.repo_name }}/{{ cookiecutter.pkg_name }}/config/settings/common.py

```diff
@@ -135,15 +134,30 @@ class Common(Configuration):
         'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
         'django.contrib.messages.middleware.MessageMiddleware',
         'django.middleware.clickjacking.XFrameOptionsMiddleware',
+        'django.middleware.security.SecurityMiddleware',
     )
 
     ROOT_URLCONF = 'config.urls'
 
     WSGI_APPLICATION = 'config.wsgi.application'
 
-    TEMPLATE_DIRS = (
-        os.path.join(BaseDir.BASE_DIR, 'templates'),
-    )
+    TEMPLATES = [
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'DIRS': [os.path.join(BaseDir.BASE_DIR, 'templates'), ],
+            'APP_DIRS': True,
+            'OPTIONS': {
+                'context_processors': [
+                    'django.template.context_processors.debug',
+                    'django.template.context_processors.request',
+                    'django.contrib.auth.context_processors.auth',
+                    'django.contrib.messages.context_processors.messages',
+                    'config.context_processors.django_version',
+                ],
+                'debug': values.Value('DEBUG', environ_name='TEMPLATE_DEBUG'),
```

Comment:
I just checked the [templates documentation](https://docs.djangoproject.com/en/1.8/topics/templates/#django.template.backends.django.DjangoTemplates) which explains:

> [The `debug` option] defaults to the value of the DEBUG setting.

This means it is obsolete to set it explicitly because we usually don't want to activate template debugging when the general debugging is turned off.